CMakeLists.txt

```diff
@@ -29,6 +29,29 @@ if (WIN32)
     set(CMAKE_CONFIGURATION_TYPES Release CACHE STRING "Semicolon separated list of supported configuration types, only supports Debug, Release, MinSizeRel, and RelWithDebInfo, anything else will be ignored." FORCE)
   endif ()
 endif (WIN32)
+if ( ENABLE_CODECOVERAGE )
```

Comment:
Remove whitespaces here and in the rest of the changes since it is not in line with the style in the rest of the CMake file.